src/include/extern.h

```diff
@@ -2353,11 +2353,21 @@ static inline void __wt_upd_value_assign(WT_UPDATE_VALUE *upd_value, WT_UPDATE *
 static inline void __wt_upd_value_clear(WT_UPDATE_VALUE *upd_value);
 
 #ifdef HAVE_UNITTEST
+/* 
+ * Expose unit test functions that wrap static functions 
+ * so that those static functions can be unit tested.
+ */
+
+/* Extent list functions */
```

Comment:
`s_all` is going to blow this away (and probably pick up some style nitpicks in the rest of the code), but again that's probably something for the next PR into `develop` rather than this one.

Reply:
Ah yes, sadly, I think you're right. I'll see what `s_all` says/does after merging, and fix up whatever is required.